libraries/Examples/examples/Example4_analogRead/Example4_analogRead.ino

```diff
@@ -38,9 +38,11 @@ void loop()
 {
   digitalWrite(LED, LOW);
 
+#ifdef A3
```

Comment:
When a user selects a board without A3 (perhaps it uses A23 or some other alias -- who knows, it is up to the board designer what to call pins) this will cause the example to compile and simply drop a desired feature without a warning. In this case I believe a warning to be acceptable. It would signal to the user "your board does not have A3" as explained in the comments at the top.

Reply:
I was approaching this from a testing standpoint (verses an education one) where the rest of the example still had testing value and I wanted to fail tests on warnings.  I disable Example4 for in my test harness for boards that do not have A3.